invenio_rdm_records/resources/config.py

```diff
@@ -97,8 +97,6 @@ class RDMRecordResourceConfig(RecordResourceConfig, ConfiguratorMixin):
     # Review
     routes["item-review"] = "/<pid_value>/draft/review"
     routes["item-actions-review"] = "/<pid_value>/draft/actions/submit-review"
-    # Record communities
```

Comment:
I think this is still used here: https://github.dev/ntarocco/invenio-rdm-records/blob/230d77ed54eaf282916a1d570c5c93dc06afa6f4/invenio_rdm_records/resources/resources.py#L174

Reply:
it was a leftover: the service config has been split, it is now [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/resources/config.py#L274).